src/main/java/org/team1540/robot2022/Robot.java

```diff
@@ -67,16 +59,18 @@ public void robotPeriodic() {
         // block in order for anything in the Command-based framework to work.
         CommandScheduler.getInstance().run();
 
-        // Update the limelight's custom SmartDashboard values
-        limelight.updateSmartDashboardValues();
+        // Update the robotContainer.limelight's custom SmartDashboard values
```

Comment:
```suggestion
        // Update the limelight's custom SmartDashboard values
```
Seems like find and replace included this one too